src/components/data/PowerBIReport/components/RlsErrorMessage/index.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import * as styles from './styles.less';
+import { Report } from '@equinor/fusion/lib/http/apiClients/models/report/';
+import {
+    MarkdownViewer,
+    Spinner,
+    Button,
+    PersonCard,
+    Accordion,
+    AccordionItem,
+    useElevationClassName,
+} from '@equinor/fusion-components';
+import { useCurrentUser, useApiClients } from '@equinor/fusion';
+import classNames from 'classnames';
+
+type RlsErrorMessageProps = {
+    report: Report;
+};
+
+const RlsErrorMessage: React.FC<RlsErrorMessageProps> = ({ report }) => {
+    const [isFetching, setIsFetching] = React.useState<boolean>(true);
+    const [requirements, setRequirements] = React.useState<string | null>(null);
+    const [description, setDescription] = React.useState<string | null>(null);
+    const [noAccessMessage, setNoAccessMessage] = React.useState<string | null>(null);
+    const [isAccessControlDescriptionsOpen, setAccessControlDescriptionOpen] = React.useState(
+        false
+    );
+
+    const reportApiClient = useApiClients().report;
+    const user = useCurrentUser();
+    const timeStamp = React.useMemo(() => new Date().toString(), []);
+
+    React.useEffect(() => {
+        getRlsRequirements();
+    }, []);
+
+    const getRlsRequirements = async () => {
+        setIsFetching(true);
+        const fetchedRequirements = await reportApiClient.getRlsRequirements(report.id);
+        const fetchedNoAccessMessage = await reportApiClient.getAccessDescription(report.id);
+        const fetchedDescriptions = await reportApiClient.getDescription(report.id);
+
+        setNoAccessMessage(fetchedNoAccessMessage?.data || null);
+        setDescription(fetchedDescriptions?.data || null);
+        setRequirements(fetchedRequirements?.data || null);
+        setIsFetching(false);
+    };
+
+    if (isFetching)
+        return <Spinner title={'Retrieving report access specifications '} floating centered />;
```

Comment:
the effect is only triggered on initial load, changing report will not fetch associated requirements.
there is no try/catch/final on awaited promises, so any api errors will always show spinner.